routes/favicon.coffee

```diff
@@ -0,0 +1,28 @@
+request = require 'request'
+fs = require 'fs'
+cachefile = "favicon.ico"
+
+favicon = (req, res, next) ->
+  try
```

Comment:
Please match the style of the other files. In particular here, tabs for indentation.